server/models/subscription.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (sequelize, DataTypes) => {
+    const Subscription = sequelize.define("Subscription", {
+        type: DataTypes.ENUM("Subscription"),
+        field: DataTypes.STRING,
+        topic: DataTypes.STRING,
+        filter: DataTypes.JSON
+    });
+
+    Subscription.associate = models => {
```

Comment:
We do the associations slightly different, look here: https://github.com/aerogear/data-sync-ui/blob/master/server/models/index.js#L12

Reply:
oops, sorry I'll fix that asap